vueapp/src/components/Game.vue

```diff
@@ -83,4 +86,11 @@ function moveGameToCourt() {
   selectedCourt.value.game = game.value;
   gameStore.removeFromOnDeck(game.value);
 }
+
+function completeGame(court) {
```

Comment:
Just reference the existing court variable
```suggestion
function completeGame() {
```